includes/ajax-functions.php

```diff
@@ -944,9 +946,10 @@ function edd_ajax_add_order_item() {
 		if ( false === $download['price_id'] ) {
 			$amount = floatval( $d->get_price() );
 		} else {
-			$prices = wp_filter_object_list( $d->get_prices(), array( 'index' => $download['price_id'] ) );
```

Comment:
As a note here, since `index` is not a properly within the array, the `wp_filter_object_list` does not find a `price_id`. The `index` is used when re-ordering the price IDs to display in a different order. The Array keys are used to determine the price ID.

Reply:
To be clear here, `index` is only used if you reorder the prices, in order to maintain their original state.

Basically `index` is not the `price_id` it is the order they should be displayed in. The `price_id` is the array key, which is what we want here, not the `display order` value.

![image](https://user-images.githubusercontent.com/1390124/44696090-ef017a80-aa2a-11e8-8b42-32e77bb19a1c.png)